bioconda_utils/cli.py

```diff
@@ -35,36 +37,62 @@
 @arg('--force',
      help='Force building the recipe even if it already exists in '
      'the bioconda channel')
-@arg('--docker', nargs='?', metavar='URL', const='unix://var/run/docker.sock',
-     help='Build packages in docker container (continuumio/conda_builder_linux).')
+@arg('--docker', action='store_true',
+     help='Build packages in docker container.')
 @arg('--loglevel', help="Set logging level (debug, info, warning, error, critical)")
+@arg('--mulled-test', action='store_true', help="Run a mulled-build test on the built package")
+@arg('--build_script_template', help='''Filename to optionally replace build
```

Comment:
It actually works the same (at least with the `@arg` decorator). The benefit of the multiline is easier editing without having to manually re-wrap.